NWNXLib/NWVariant.cpp

```diff
@@ -0,0 +1,28 @@
+#include "NWVariant.hpp"
+#include "Utils.hpp"
+
+namespace NWNXLib
+{
+
+std::string NWVariant::toString() const
```

Comment:
The header file is already 4x the cpp file, and this is almost a trivial function. I'd just collapse it into the header and get rid of the cpp entirely - we don't gain much from this, it just splits code further.

Reply:
Will do.